src/components/Module.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div>
+    <h1>Module number {{ moduleId }}</h1>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Module',
+  props: {
+    moduleId: String
+  }
+}
+</script>
+
+<style scoped>
+
+</style>
```

Comment:
Created a module component but this might not be needed (I'm not sure yet). I just wanted to use the route to say what module we're in.